modern/src/reports/ChartType.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { FormControl, Select, MenuItem } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import t from '../common/localization';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    minWidth: 160,
+  },
+}));
+
+const ChartType = ({ type, setType }) => {
+  const classes = useStyles();
+
+  return (
+    <FormControl className={classes.formControl}>
+      <Select value={type} onChange={e => setType(e.target.value)} >
```

Comment:
I think we should start with a smaller subset for now. I don't think it's a good way to maintain this list specifically for charts. Maybe we can have a list somewhere else as a more generic list that can be converted into select.

Reply:
Removed this component and added chartTypes list and then created a select using chartTypes.